### ISSUE TITLE
Unable to read tool run details with screenreader

### ISSUE THREAD

**Issue by rzhao271** [microsoft/vscode#247645]
Testing #247533

On Linux with a Python file open and with a venv activated, I asked GPT-4.1 "What Python version am I using?"
After expanding the tool run details to see the input and output, I noticed that orca did not read any of it.
